middie.js

```diff
@@ -64,12 +77,17 @@ function middie (complete) {
       } else {
         const middleware = middlewares[i]
         const fn = middleware.fn
-        if (!middleware.path) {
-          fn(req, res, that.done)
-        } else if (middleware.wildcard && pathMatchWildcard(url, middleware.path)) {
-          fn(req, res, that.done)
-        } else if (middleware.path === url || (typeof middleware.path !== 'string' && middleware.path.indexOf(url) > -1)) {
+        const regexp = middleware.regexp
```

Comment:
I think here and above we could just use `var`.

Reply:
updated